pkg/rules/handlers.go

```diff
@@ -0,0 +1,267 @@
+package rules
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/http/types"
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/go-chi/chi/v5"
+)
+
+type API struct {
+	service ServiceInterface
+
+	logger logging.LoggerInterface
+}
+
+func (a *API) RegisterEndpoints(mux *chi.Mux) {
+	mux.Get("/api/v0/rules", a.handleList)
+	mux.Get("/api/v0/rules/{id}", a.handleDetail)
+	mux.Post("/api/v0/rules/{id}", a.handleCreate)
```

Comment:
ID shouldn't be picked by the URL, it comes either from the payload (not great) or we create a random one from the data we ve been given, like a uuidv5

Reply:
I've seen only a couple of examples for oathkeeper access rules. The convention for the ID (for example: zinc-k8s:deny) seems to be closely tied to the app and the function of the rule. I'll check with Natalia. If it's unconventional to have random ids than I would rather not implement it, but I'll definitely take the ID url parameter out.

Reply:
No ID parameter for the post handler. The ID is taken out from the payload object.